src/platform/packages/shared/shared-ux/chrome/navigation/src/ui/components/navigation_item_open_panel.tsx

```diff
@@ -8,100 +8,33 @@
  */
 
 import React, { useCallback, type FC } from 'react';
-import { i18n } from '@kbn/i18n';
 import classNames from 'classnames';
-import { css } from '@emotion/css';
-import {
-  EuiButtonIcon,
-  EuiFlexGroup,
-  EuiFlexItem,
-  EuiListGroup,
-  EuiListGroupItem,
-  type EuiThemeComputed,
-  useEuiTheme,
-  transparentize,
-  useIsWithinMinBreakpoint,
-  EuiButton,
-} from '@elastic/eui';
+import { css } from '@emotion/react';
+import { transparentize, EuiButton } from '@elastic/eui';
 import type { ChromeProjectNavigationNode } from '@kbn/core-chrome-browser';
-import { useNavigation as useServices } from '../../services';
 import { isActiveFromUrl } from '../../utils';
 import type { NavigateToUrlFn } from '../../types';
 import { usePanel } from './panel';
 
-const getStyles = (euiTheme: EuiThemeComputed<{}>) => css`
-  * {
-    // EuiListGroupItem changes the links font-weight, we need to override it
-    font-weight: ${euiTheme.font.weight.regular};
-  }
-  &.sideNavItem:hover {
-    background-color: transparent;
-  }
-  &.sideNavItem--isActive:hover,
-  &.sideNavItem--isActive {
-    background-color: ${transparentize(euiTheme.colors.lightShade, 0.5)};
-    & * {
-      font-weight: ${euiTheme.font.weight.medium};
-    }
-  }
-`;
-
-const getButtonStyles = (euiTheme: EuiThemeComputed<{}>, isActive: boolean) => css`
-  background-color: ${isActive ? transparentize(euiTheme.colors.lightShade, 0.5) : 'transparent'};
-  transform: none !important; /* don't translateY 1px */
-  color: inherit;
-  font-weight: inherit;
-  padding-inline: ${euiTheme.size.s};
-  & > span {
-    justify-content: flex-start;
-    position: relative;
-  }
-  & .euiIcon {
-    position: absolute;
-    right: 0;
-    top: 0;
-    transform: translateY(50%);
-  }
-`;
-
 interface Props {
   item: ChromeProjectNavigationNode;
   navigateToUrl: NavigateToUrlFn;
   activeNodes: ChromeProjectNavigationNode[][];
 }
 
-export const NavigationItemOpenPanel: FC<Props> = ({ item, navigateToUrl, activeNodes }: Props) => {
-  const { euiTheme } = useEuiTheme();
+export const NavigationItemOpenPanel: FC<Props> = ({ item, activeNodes }: Props) => {
   const { open: openPanel, close: closePanel, selectedNode } = usePanel();
-  const { isSideNavCollapsed } = useServices();
-  const { title, deepLink, children } = item;
+  const { title, deepLink } = item;
   const { id, path } = item;
-  const href = deepLink?.url ?? item.href;
-  const isNotMobile = useIsWithinMinBreakpoint('s');
-  const isIconVisible = isNotMobile && !isSideNavCollapsed && !!children && children.length > 0;
-  const hasLandingPage = Boolean(href);
   const isExpanded = selectedNode?.path === path;
-  const isActive = hasLandingPage ? isActiveFromUrl(item.path, activeNodes) : isExpanded;
```

Comment:
Previous code had a bug: `isActiveFromUrl` was only called if the nav item had a landing page. This was broken for the parent nav item buttons that are remaining in this PR.